javascript/ql/src/semmle/javascript/frameworks/Micro.qll

```diff
@@ -0,0 +1,118 @@
+/**
+ * Provides a model of the `micro` NPM package.
+ */
+
+private import javascript
+
+private module Micro {
+  private import DataFlow
+
+  /**
+   * A node that should be interpreted as a route handler, to use as starting
+   * point for back-tracking.
+   */
+  Node microRouteHandlerSink() {
+    result = moduleMember("micro", "run").getACall().getLastArgument()
+    or
+    result = moduleImport("micro").getACall().getArgument(0)
+  }
+
+  /** Gets a data flow node referring to a thing. */
```

Comment:
A "thing"? Is that a placeholder?
(ditto below)

Reply:
Can't say it isn't technically correct. 😄